iphone/Classes/TiUIListView.m

```diff
@@ -871,6 +861,7 @@ -(void)setSearchView_:(id)args
             [[searchViewProxy searchBar] setPlaceholder:@"Search"];
         }
         self.searchString = [[searchViewProxy searchBar] text];
+        [self initSearchController:self];
```

Comment:
Not a problem . Every time searchViewController is released and new searchResultController is created.